libs/entry-components/styles/partials/vendors/overrides/rich-text/_general.scss

```diff
@@ -0,0 +1,12 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+
+.enigmatry-redactor-content {
+	.rx-button-icon svg {
+		fill: rgb(0 0 0 / .54);
```

Comment:
Angular Material’s built-in palettes don’t provide direct shades of black (rgb(0, 0, 0)) with specific opacities like .38 or .54. The existing palettes, such as $grey-palette, offer different shades of gray but don’t include true black with those exact transparency levels. Using rgba in this way allows us to achieve the exact colors and opacity levels without needing to rely on Angular Material’s palette system for custom black opacities.